fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/client/GitHubClient.java

```diff
@@ -43,10 +43,7 @@
   List<Member> getOrgMembers(
       @Path("org") String org, @Query("page") int page, @Query("per_page") int paginationValue);
 
-  @GET("/teams/{idTeam}/members")
+  @GET("/orgs/{org}/teams/{idTeam}/members")
   List<Member> getMembersOfTeam(
-      @Path("idTeam") Long idTeam, @Query("page") int page, @Query("per_page") int paginationValue);
-
-  @GET("/teams/{idTeam}/memberships/{username}")
```

Comment:
this function does not seem to be used at all

Reply:
https://github.com/spinnaker/fiat/blob/872873672b5910cb86f24b797eab90066a0ee531/fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java#L230
NOT dug through entire code, but seemed to be there?

Reply:
this is calling https://github.com/spinnaker/fiat/blob/836fac81cc15f1cc1b748e4295ad749b919bce43/fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java#L315 - but could be wrong as I haven't touched java in a while

Reply:
Pretty sure `getUserTeams`, `isMemberOfOrganization` & `isMemberOfTeam` are unused and can be cleaned up.